### ISSUE TITLE
Add an option to edit the keyboard keys

### ISSUE THREAD

**Issue by ZephyrWeiss** [luern0313/WindSong-Lyre-Genshin-Impact#27]
Previously the piano in Identity V (another Chinese game) was like the Lyre and only had 3 octaves without black keys. But now they added the black keys and this program is one of the best by being able to press the keys for a long time. An option or configuration file to edit the keyboard keys would be great, making this program universal for all games, pages, software with piano, etc.

**Comment by luern0313** [comment]
Could you let me know the keyboard keys that correspond to the added black keys in Identity V?

**Comment by ZephyrWeiss** [comment]
Of course, I have edited a photo and created a map to make it easier for you.
![Freepiano 2 - IDV map](https://github.com/luern0313/WindSong-Lyre-Genshin-Impact/assets/107342572/2ea0d980-e098-4f9c-a29a-6a9f4f5325c2)
![IdentityV - Piano](https://github.com/luern0313/WindSong-Lyre-Genshin-Impact/assets/107342572/2d625ad9-5e11-4a6f-b91f-84f1421456a6)
![IDV piano keyboard](https://github.com/luern0313/WindSong-Lyre-Genshin-Impact/assets/107342572/cb0bc5cb-7b3c-4141-ab0b-f30534031af5)

**Comment by ZephyrWeiss** [comment]
> Could you let me know the keyboard keys that correspond to the added black keys in Identity V?

Of course, I have edited a photo and created a map to make it easier for you:
![Freepiano 2 - IDV map](https://github.com/luern0313/WindSong-Lyre-Genshin-Impact/assets/107342572/30150bc7-002d-409f-8efe-479b2c08a8b5)
![IdentityV - Piano](https://github.com/luern0313/WindSong-Lyre-Genshin-Impact/assets/107342572/dada60d8-a8a4-4d2b-8af7-29a5d535d42a)
![IDV piano keyboard](https://github.com/luern0313/WindSong-Lyre-Genshin-Impact/assets/107342572/bf354729-3df3-4069-91b1-2c2b036f6b36)

**Comment by luern0313** [comment]
Ok, I'll see if I can develop that feature when I find some time to give it a shot.
